nibiru/assetlist.json

```diff
@@ -29,6 +29,45 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nibiru/images/nibiru.svg"
         }
       ]
+    },
+    {
+      "description": "uoprek",
+      "denom_units": [
+        {
+          "denom": "tf/nibi149m52kn7nvsg5nftvv4fh85scsavpdfxp5nr7zasz97dum89dp5qkyhy0t/uoprek",
+          "exponent": 0
+        }
+      ],
+      "base": "uoprek",
```

Comment:
this base denom is not defined in denom_units. I'm thinking base should be "tf/nibi149m52kn7nvsg5nftvv4fh85scsavpdfxp5nr7zasz97dum89dp5qkyhy0t/uoprek"

Reply:
Resolved